llvm/test/Transforms/Inline/AArch64/sme-pstatesm-attrs.ll

```diff
@@ -2,70 +2,71 @@
 ; RUN: opt < %s -mtriple=aarch64-unknown-linux-gnu -mattr=+sme -S -passes=inline | FileCheck %s
 
 declare void @inlined_body() "aarch64_pstate_sm_compatible";
+declare i32 @llvm.vscale()
 
 ; Define some functions that will be called by the functions below.
 ; These just call a '...body()' function. If we see the call to one of
 ; these functions being replaced by '...body()', then we know it has been
 ; inlined.
```

Comment:
Does this comment need to be updated now that `@inlined_body()` has been replaced with `@llvm.vscale.i32()`?

Reply:
Good spot, I've rephrased it.